panels/templates/panel_applications/guest.html

```diff
@@ -0,0 +1,81 @@
+{% extends "preregistration/preregbase.html" %}
+{% import 'panel_macros.html' as panel_macros %}
+{% block title %}Panel Application{% endblock %}
+{% block backlink %}{% endblock %}
+{% block content %}
+  {% include "panels_common.html" %}
+
+  {# This needs to be placed outside the <form> tag or we get validation errors #}
```

Comment:
I really wanted to have the Other Panelists section self-contained, but when it was in the new template I made, it threw validation errors because the input tags were considered part of the form. :(

Reply:
You could keep this with the Other Panelists section, and move it under the `<body>` element on document load.

```javascript
$(function () {
    $('#panelist_form_template').appendTo($('body'));
});
```